docs/source/integration-guide.rst

```diff
@@ -0,0 +1,189 @@
+Integration Guide
+=================
+
+This guide explains how to use ``astrolabe`` to add Atlas Planned Maintenance testing
+to your driver.
+
+
+--------
+Overview
+--------
+
+Driver projects looking to integrate this framework into their project codebase should follow the following steps:
+
+#. Implement a Workload Executor script for their driver project.
+#. Implement a Driver Installation script for their driver project.
+#. Update the Evergreen configuration of the ``drivers-atlas-testing`` project.
+
+
+We will walk through each of these steps in detail in the following sections.
+
+Setting up
+----------
+
+#. Start by cloning the ``drivers-atlas-testing`` repository and creating a new branch.
```

Comment:
Actually, I forgot a step here (to fork the repo first, then clone, then branch). In that light, the code-samples wont exactly be copy-pastable since we have no way to pre-empt the GH usernames of folks who will be using this, but I have added samples with placeholders nonetheless.